### ISSUE TITLE
Switch enter button icon to play button when in a command mode

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Android#160]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

One of the usability features of Scribe is that the enter key switches color and its icon when the user is in a command mode to enter a command. As of now when a user presses the Scribe key and then a command button they're shown the command bar and they'll then type in the command bar as well. The enter key is then Scribe blue, but we also want that its icon switches to a play button. The play button icon should be filled in - i.e. not the just the boarder. An example of the interface in Scribe-iOS can be seen in the Figma designs [here](https://www.figma.com/design/c8945w2iyoPYVhsqW7vRn6/scribe_public_designs?node-id=403-9450&t=QznVUA7nMwE5CZge-1) (the interface over translation) and below:

![iPhone 14 White Minimal](https://github.com/user-attachments/assets/912e4f2c-86e0-4b4b-908f-b9708c3eff0c)

### Contribution

This is a great good first issue! Happy to review or support someone wanting to get into Scribe-Android development as needed 😊

**Comment by KekmaTime** [comment]
@andrewtavis hey i would love to work on this issue as part of hacktoberfest

**Comment by andrewtavis** [comment]
Sure thing, @KekmaTime! Let us know if there's anything we can do to support 😊

**Comment by andrewtavis** [comment]
Hey @KekmaTime 👋 Please let us know if you need any support here :)

**Comment by Linfye** [comment]
@KekmaTime Do you still want to work on this issue? Can I participate or collaborate? 😊

cc @andrewtavis

**Comment by andrewtavis** [comment]
Thanks for passing it off, @KekmaTime, and thanks for picking it up, @Linfye! 🚀

**Comment by andrewtavis** [comment]
Wouldn't it be setting the key icon and then reloading the keyboard whenever the keyboard is in a command mode? What have you tried so far, @Linfye?

**Comment by Linfye** [comment]
I tried to add a judgment in the keyboard build helper, and then dynamically update the value of isCommandMode in the SimpleKeyboardIME, but the keyboard did not update the icon after this value was updated.

```
if (key.code == KEYCODE_ENTER) {
    val enterResourceId = if (isCommandMode) {
        R.drawable.play_button
    } else {
        when (mEnterKeyType) {
            EditorInfo.IME_ACTION_SEARCH -> R.drawable.ic_search_vector
            EditorInfo.IME_ACTION_NEXT, EditorInfo.IME_ACTION_GO -> R.drawable.ic_arrow_right_vector
            EditorInfo.IME_ACTION_SEND -> R.drawable.ic_send_vector
            else -> R.drawable.ic_enter_vector
        }
    }
    key.icon = context.resources.getDrawable(enterResourceId, context.theme)
}
```

**Comment by andrewtavis** [comment]
And did you recreate the keyboard UI after the variable was set? In iOS we need to change the value of the key icon and then recreate the entire keyboard UI.

**Comment by andrewtavis** [comment]
Also feel free to open a PR and we can look at it, @Linfye! 😊

**Comment by andrewtavis** [comment]
Closed by #251 😊 Thanks for the amazing work here and for being such a great collaborator on Scribe-Android, @Linfye!
